torch_geometric/loader/neighbor_loader.py

```diff
@@ -152,6 +165,8 @@ def __init__(
             raise TypeError(f'NeighborLoader found invalid type: {type(data)}')
 
     def _sparse_neighbor_sample(self, index: Tensor):
+        if len(self.num_neighbors) == 0:
```

Comment:
What happens if we run the sampler on empty dictionaries?

Reply:
We can pass an emtpy list of `num_neighbors` to `torch_sparse.neighbor_sample`. But the issue is that `colptr` and `row` can't be empty tensors, that throws a segfault.

Reply:
Yeah, see my reply above.